code/__DEFINES/_tick.dm

```diff
@@ -0,0 +1,10 @@
+#define TICK_LIMIT_RUNNING 80
+#define TICK_LIMIT_TO_RUN 78
+#define TICK_LIMIT_MC 70
+#define TICK_LIMIT_MC_INIT_DEFAULT 98
+
+#define TICK_USAGE world.tick_usage //for general usage
+#define TICK_USAGE_REAL world.tick_usage    //to be used where the result isn't checked
```

Comment:
why two defines with same value

Reply:
TICK_USAGE_REAL is for purely informative checks, one day I may (See project) add side effects to TICK_USAGE